docs/source/quickstart.rst

```diff
@@ -84,7 +80,7 @@ To do this, you can either
 
 1. Install Chaco and its :ref:`dependencies` from `PyPI
    <http://pypi.python.org/pypi>`_ using `easy_install
-   <http://packages.python.org/distribute/easy_install.html>`_ (part of
```

Comment:
Sorry, I meant EPD. (I didn't realize that ETS is used for Enthought Training Services and not Enthought Tool Suite) As in, instead of pointing people towards EPD as a way to install Chaco, point them towards Canopy as a way to install and use it, given that Canopy is a superset (of sorts) to EPD.